src/Tizen.NUI.Components/Controls/Tab.cs

```diff
@@ -591,19 +591,6 @@ private void UpdateItems()
             }
         }
 
-        private void CreateUnderLineAttributes()
```

Comment:
Is it deleted because We initialize this in TabStyle class?

Reply:
No, it is a history unuseful api, so I removed it.